torch/jit/__init__.py

```diff
@@ -143,9 +141,7 @@ def forward(self, x):
             extra_files['foo.txt'] = 'bar'
             torch.jit.save(m, 'scriptmodule.pt', _extra_files=extra_files)
     """
-    if isinstance(f, str) or \
-            (sys.version_info[0] == 2 and isinstance(f, unicode)) or \
-            (sys.version_info[0] == 3 and isinstance(f, pathlib.Path)):
+    if isinstance(f, str) or isinstance(f, pathlib.Path):
```

Comment:
Minor nit, but not merge blocking is that `isinstance` can actually take in an iterable for the second argument so this could be rewritten to
```suggestion
    if isinstance(f, (str, pathlib.Path)):
```